code/game/atoms.dm

```diff
@@ -85,8 +85,8 @@
 		canSmoothWith = typelist("canSmoothWith", canSmoothWith)
 
 	if(datum_outputs)
-		for(var/A in datum_outputs)
-			A = SSoutputs.outputs[A]
+		for(var/i in 1 to length(datum_outputs))
+			datum_outputs[i] = SSoutputs.outputs[datum_outputs[i]]

```

Comment:
datum_outputs[SSoutputs.outputs[datum_outputs[i]]] = 0
0 can be replaced for a list later if we will store more than just cooldown